docs/configuration/plugins.md

```diff
@@ -33,3 +33,21 @@ Note that a plugin must be listed in `PLUGINS` for its configuration to take eff
 
 ---
 
+## PLUGINS_CONFIG
+
+Default: Empty
+
+This parameter holds configuration settings for how the individual plugins are displayed in the plugins table under Admin->System->Plugins.  Declaring `hidden` as True will make it so the plugin is not shown in the table.  Declaring `disabled` will make it show the plugin in the table, but it won't be linked to the details and installation page for the plugin. An example configuration is shown below:
```

Comment:
I worry that using the term "disabled" here invites confusion with _actually_ disabling a plugin, rather than merely removing its link in the catalog. Trying to think of a better term.

Reply:
Maybe we could denote these plugins as "unavailable" (with the presumption they are not installed).

Reply:
changed it to unlinked